portality/tasks/redis_huey.py

```diff
@@ -1,4 +1,13 @@
-from huey import RedisHuey
+from huey import RedisHuey, crontab
 from portality.core import app
 
 main_queue = RedisHuey('doaj', host=app.config['HUEY_REDIS_HOST'], port=app.config['HUEY_REDIS_PORT'])
+
+def schedule(action):
+    cfg = app.config.get("HUEY_SCHEDULE", {})
```

Comment:
If I do schedule('lalalala') it's going to return * * * * * and run every minute. I think this is going to be a bit misleading and could result in unintentional runs (e.g. a typo of "shcedule_sitemap" means it will now run every minute rather than every day"). Could you make this function actually fail if the requested schedule does not exist?

So simply on lines 7 and 8

cfg = app.config['HUEY_SCHEDULE']  # you're not going anywhere without any configs if you're calling this function
action_cfg = cfg[action]  # the action you've passed as an arg to the schedule() function has to have a matching schedule, else fail